install_config/aggregate_logging.adoc

```diff
@@ -416,24 +416,84 @@ The deployer creates an ephemeral deployment in which all of a pod's data is
 lost upon restart. For production usage, add a persistent storage volume to each
 Elasticsearch deployment configuration.
 
-The following example specifies a volume for an Elasticsearch replica (using a
-xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[PersistentVolumeClaim]):
+The best-performing volumes are local disks, if it is possible to use
+them. There are some complications with doing so.
 
+. The relevant service account must be given the privilege to mount and edit a local volume, as follows:
++
 ====
 ----
-$ oc volume dc/logging-es-rca2m9u8 \
+$ oadm policy add-scc-to-user privileged  \
```

Comment:
The kubelet (when a pod is using host namespaces) or docker should be performing a relabeling of the volume when it can.  It uses the docker opts to pass in the selinux context that is being used.  If that isn't working or this is a different use case then we can figure out what is different.  cc @pmorie who is very familiar with the selinux code for volumes